pkg/minishift/util/set_driver_options_from_env.go

```diff
@@ -0,0 +1,46 @@
+package util
+
+import (
+	"github.com/docker/machine/libmachine/drivers"
+	"reflect"
+	"github.com/docker/machine/libmachine/mcnflag"
+	"os"
+)
+
+func SetDriverOptionsFromEnvironment(d drivers.Driver) error {
+	supportedFlags := d.GetCreateFlags()
+
+	checkFlags := &drivers.CheckDriverOptions{
+		FlagsValues: map[string]interface{}{},
+		CreateFlags: supportedFlags,
+	}
+
+	for _, flag := range supportedFlags {
+		switch reflect.TypeOf(flag).String() {
```

Comment:
@hferentschik , the underline type for `flag` can be `mcnflag.StringFlag`, `mcnflag.IntFlag` etc. or multiple.

as `flag` is a interface type here.